### PR TITLE
Create claims for authors of fallback posts

### PR DESCRIPTION
**Pull request by majelbstoat**
Hello @kylehg, @mikkot, 

Please review the following commits I made in branch 'jamie-claim-api'.

d6b91bccaf32d83b6965eafdfd5e75b62cd5d22d (2015-12-02 20:20:29 -0800)
Create claims for authors of fallback posts

R=@kylehg
R=@mikkot

### CODE REVIEW

**Comment by majelbstoat** [comment]
Don't merge this until the claim endpoint is ready, but once it's done, this should hopefully work straight away. You can test this by creating a local wordpress post by an author with no integration token, then running the migration tool. 

NB: This patch uses `md5` as the body parameter per my comment @mikkot's PR.

**Comment by mikkot** [comment]
:bow:

File: lib/medium-admin.php [hunk 1]
```diff
@@ -764,13 +765,26 @@ public static function cross_post($post, $medium_post, $medium_user) {
...
       "Content-Type" => "application/json"
     ));
   }
 
   /**
+   * Creates a post claim for the original author of a post.
+   */
+  public static function create_post_claim(Medium_Post $medium_post, Medium_User $medium_user, $user) {
+    $body = array(
+      "md5" => md5($user->user_email)
```
Comment (mikkot) [inline lib/medium-admin.php]: We should probably downcase the email to make the comparison case-insensitive across WP/Medium

Comment (majelbstoat) [inline lib/medium-admin.php]: Good idea. Not technically to spec, but will probably save issues.

Comment (mikkot) [inline lib/medium-admin.php]: Yeah - don't worry about this though, we can tweak it.

Comment (majelbstoat) [inline lib/medium-admin.php]: Done!

**Comment by mikkot** [comment]
LG, merging on behalf of @majelbstoat

**Pull request merged**